lib/ae_page_objects/version.rb

```diff
@@ -1,3 +1,3 @@
 module AePageObjects
-  VERSION = '2.0.0'.freeze
+  VERSION = '2.1.0.beta1'.freeze
```

Comment:
Please revert this change. The release process for this gem follows the common practice of releasing off of master.